WTWSMS/decisions/WtWSMS_form_titles_decisions.txt

```diff
@@ -390,7 +390,7 @@ decisions = {
 }
 
 decisions = {
-	this_is_sparta = {
+	this_is_sparta = { # Edits and commentary courtesy of Loup
```

Comment:
Yeah, although you didn't implement all I said and added new things I disagree on.